js/adapt-tincan.js

```diff
@@ -293,6 +326,21 @@ define(function(require) {
       block.getChildren().each(function (child) {
         child.set('_isComplete', true);
       }, this);
+    },
+    
+    validateParams: function() {
+        var isValid = true;
+        if (!actor || typeof actor != 'object') {
+            console.log('actor object not valid' + typeof actor);
```

Comment:
I'm not sure we should be writing to the console (perhaps only if we're in “debug” mode?) If we do, this string needs a space at the end so it doesn't look like 'actor object not validarray' (for example).